packetbeat/sniffer/sniffer.go

```diff
@@ -133,6 +137,9 @@ func New(testMode bool, _ string, decoders Decoders, interfaces []config.Interfa
 				if iface.BufferSizeMb <= 0 {
 					iface.BufferSizeMb = 24
 				}
+				if iface.MetricsInterval == 0 {
+					iface.MetricsInterval = 5 * time.Second
+				}
```

Comment:
This needs to be checked for negative values, otherwise we'll panic when we create the polling ticker. Either make the condition here <= 0 and change the docs to say <=0 will default to 5s, or add a validation check in the config (probably also needs a doc change to say it must be non-negative).